File: ProjectFinalAdminBD/src/java/dominio/Cliente.java
package dominio;

/**
 * @author emfam
 */
public class Cliente {

    private int cedula;
    private String nombre;
    private String direccion;
    private int telefono;
    private String ocupacion;
    private String correo;
    
    public Cliente(){}

    public Cliente(int cedula, String nombre, String direccion, int telefono, String ocupacion, String correo) {
        this.cedula = cedula;
        this.nombre = nombre;
        this.direccion = direccion;
        this.telefono = telefono;
        this.ocupacion = ocupacion;
        this.correo = correo;
    }

    public int getCedula() {
        return cedula;
    }

    public void setCedula(int cedula) {
        this.cedula = cedula;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    public int getTelefono() {
        return telefono;
    }

    public void setTelefono(int telefono) {
        this.telefono = telefono;
    }

    public String getOcupacion() {
        return ocupacion;
    }

    public void setOcupacion(String ocupacion) {
        this.ocupacion = ocupacion;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    @Override
    public String toString() {
        return "Cliente{" + "cedula=" + cedula + ", nombre=" + nombre + ", direccion=" + direccion + ", telefono=" + telefono + ", ocupacion=" + ocupacion + ", correo=" + correo + '}';
    }
    
}

File: ProyectoBD2021/domain/Cliente.php
<?php
	
	class Cliente
	{

		private $cedula;
		private $nombre;
        private $direccion;
        private $telefono;
        private $ocupacion;
        private $correo;

		function __construct($cedula,$nombre,$direccion,$telefono,$ocupacion,$correo)
		{
			$this->cedula = $cedula;
			$this->nombre = $nombre;
            $this->direccion = $direccion;
            $this->telefono = $telefono;
            $this->ocupacion = $ocupacion;
            $this->correo = $correo;
		}


		public function getCedula()
		{
			return $this->cedula;
		}

		public function getNombre()
		{
			return $this->nombre;
		}

        public function getDireccion()
        {
            return $this->direccion;
        }

        public function getTelefono()
        {
            return $this->telefono;
        }

        public function getOcupacion()
        {
            return $this->ocupacion;
        }

        public function getCorreo()
        {
            return $this->correo;
        }

	}
 ?>